cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -74,6 +71,23 @@
 #define CONFIG_TCP_OVERSIZE_MSS 1
 #define LL_ALIGN(s)             (((uint32_t)s + 3) & 0xfffffffcU)
 
+/**
+ * The SDK interface of the WiFi module uses the lwIP `pbuf` structure for
+ * packets sent to and received from the WiFi interface. For compatibility
+ * reasons with the binary SDK libraries we need to incclude the SDK lwIP
```

Comment:
s/incclude/include

Reply:
> Again I can look at the code and run through testing but it probably won't that attention until 15.01.2020 to 30.01.2020.

@MrKevinWeiss That's absolutely fine. I don't plan any bigger change in these parts of code in next weeks so that it will be easy to keep it consistent with current master.

Reply:
Ok, thanks. `codespell` didn't find this. Lets hope that the change will not be lost on next rebase.